templates/account/purchases/payments.html

```diff
@@ -30,7 +30,7 @@
           <a href="{{ url_for('payments.stripe_cancel', payment_id=p.id) }}" class="btn btn-warning">Cancel</a>
           {% endif -%}
 
-          {% if feature_enabled('REFUND_REQUESTS') and p.state == "paid" %}
+          {% if REFUND_STATE != "off" and p.state in ["paid", "partrefunded"] %}
```

Comment:
Checking the change here: previously it was only the case that admins could partially refund a purchase -- but now users will be able to do that, and that means that `partrefunded` purchases should be included (for further refund requests?) here.

Reply:
yes, the intention is that users will be able to refund, e.g. 3 out of 5 tickets

Reply:
Ok, that part I understood; but specifically here, the idea seems to be that users can request another refund from an already-partly-refunded purchase?

1. User purchases 5 tickets; status = `paid`; this template displays the refund request form (OK).
1. User selects refunds for 3 of 5 tickets; status = `partrefunded`; this template continues to display the refund request form (is that OK?  I think so, probably.  but this is the aspect I'm wondering about).

Reply:
I don't see why it would be a problem? Probably a question for @russss

Reply:
It's fine, it was more annoying in the past when refunds required more manual work.